web/controllers/courseController/index.ts

```diff
@@ -2773,7 +2773,7 @@ export async function groupMailHandler(req: Request, res: Response) {
             && req.params.subid != undefined
             && typeof req.body.subject == "string"
             && typeof req.body.body == "string") {
-            status = await groupMail(res.locals.user, Number.parseInt(req.params.id, 10), Number.parseInt(req.params.subid, 10), req.body.subject, req.body.body);
+            status = await groupMail(res.locals.user, Number.parseInt(req.params.id, 10), Number.parseInt(req.params.subid, 10), req.body.subject, req.body.body, JSON.parse(req.body.addressees), req.files as Express.Multer.File[]);
```

Comment:
Why are the mails passed in here? What happens to the current web user app which does not pass in `req.body.addressees`?

Reply:
`req.body.addressees` aren't mails, they are user IDs. They're in there because this feature allows you to specify the participants you want to send the message to. As for the current web app, it would probably error out, but that's why there's a [frontend PR](https://github.com/corona-school/web-user-app/pull/207) which adds this feature :) We'd have to merge them into master at the same time then.